fastly/block_fastly_service_v1_logging_ftp.go

```diff
@@ -0,0 +1,268 @@
+package fastly
+
+import (
+	"fmt"
+	"log"
+
+	fst "github.com/fastly/go-fastly/fastly"
+	"github.com/hashicorp/terraform-plugin-sdk/helper/schema"
+)
+
+var ftpSchema = &schema.Schema{
+	Type:     schema.TypeSet,
+	Optional: true,
+	Elem: &schema.Resource{
+		Schema: map[string]*schema.Schema{
+			// Required fields
+			"name": {
+				Type:        schema.TypeString,
+				Required:    true,
+				Description: "The unique name of the FTP logging endpoint",
+			},
+
+			"address": {
```

Comment:
Just making note that for FTP we don't explicitly expose IPV4 or Hostname because Address includes these values.

https://developer.fastly.com/reference/api/logging/ftp/